dmoj/cptbox/tracer.py

```diff
@@ -116,11 +115,9 @@ def __init__(
         wall_time: Optional[float] = None,
     ) -> None:
         self._executable = executable
-        self.use_seccomp = security is not None and not avoid_seccomp
 
-        if self.use_seccomp and BAD_SECCOMP:
-            log.warning('Requires Linux 4.8+ to use seccomp, you have: %s', os.uname().release)
-            self.use_seccomp = False
+        if BAD_SECCOMP:
+            raise RuntimeError(f'Requires Linux 4.8+ to use seccomp, you have: {os.uname().release}')
```

Comment:
I know you didn't change this here, but what requires 4.8? Let's have that in the message, maybe "judge requires Linux..."